index_new_user.html

```diff
@@ -0,0 +1,44 @@
+<html>
+    <head>
+            <link rel="stylesheet" href="style_new_user.css">
+    </head>
+    <body>
+        <main id="main">
+            <form id="new_user_form" action="/submit-info-here">
+                <section id="personal_information">
+                    <p id="personal_information_label">New User Login</p>
+                    <section id="first_name_section">    
+                        <label id="first-name-label" class="labels">First Name</label>
+                        <input id="first-name" type="text" placeholder="First Name" required>
+                    </section>
+                    <section id="last_name_section">                    
+                        <label id="last-name-label" class="labels">Last Name</label>
+                        <input id="last-name" type="text" placeholder="Last Name" required>
+                    </section>
+                    <section id="email_section">
+                        <label id="email-label" class="labels">Email</label>
+                        <input id="email" type="email" placeholder="Email" required>
+                    </section>
+                    <section id="admin_permissions_section">            
+                        <label id="dropdown-label">Admin permissions</label>
+                        <select id="dropdown" name="admin_permissions" class="dropdown">
+                            <option disabled value>Please select an option</option>
+                            <option value="yes">Yes</option>
+                            <option value="no">No</option>
+                            <option value="Default to no">Default to no</option>

```

Comment:
Default to no should not be an option. When the form displays the "no" value should be the default value.